src/main/java/org/broadinstitute/hellbender/utils/MannWhitneyU.java

```diff
@@ -525,7 +529,7 @@ public double permutationTest(final double[] series1, final double[] series2, fi
             assert (series2End == n2);
 
             double newU = MathUtils.sum(newSeries1) - ((n1 * (n1 + 1)) / 2.0);
-            histo.increment(newU);
+            histo.increment(FastMath.round(2 * newU));
```

Comment:
Just checking, but you shouldn't need to round, is that right? `2 * newU` should always be an integer so casting or floor would also work here?

Reply:
I'm worried that with finite precision `2 * newU` would end up slightly below the correct integer, in which case casting or floor would give the wrong value.  And maybe this isn't supposed to happen, but I'm paranoid because this whole bug is from somebody's JVM doing things that the JVM isn't supposed to do.